yt_idv/shaders/ray_tracing.frag.glsl

```diff
@@ -8,6 +8,11 @@ flat in mat4 inverse_pmvm; // always cartesian
 flat in ivec3 texture_offset;
 out vec4 output_color;
 
+flat in vec4 phi_plane_le;
+flat in vec4 phi_plane_re;
+
+const float INFINITY = 1. / 0.;
+const float PI = 3.1415926535897932384626433832795;
```

Comment:
let's move both of these to the common include

Reply:
done